Formula/spider.rb

```diff
@@ -0,0 +1,18 @@
+class Spider < Formula
+  desc "small dart library to generate Assets dart code from assets folder."
+  homepage "https://github.com/BirjuVachhani/spider"
+  url "https://github.com/BirjuVachhani/spider/releases/download/1.0.1/spider-1.0.1-macos.tar.gz"
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
What do you suggest then?

Reply:
Build it from source in the formula

Reply:
Well, it is written in Dart. I tried to build it from source code but it has a dependency on `pub` and `dart2native` tools which are included in the `dart sdk`. Is it a good idea to have dart sdk loaded when trying to install?

Reply:
If you can compile that from source, yes.